titles/admin-rhdh/title-admin.adoc

```diff
@@ -10,11 +10,22 @@ The {product} is an enterprise-grade, open developer platform that you can use t
 //customer support links
 include::artifacts/snip-customer-support-info.adoc[]
 
-// install operator
-include::modules/admin/proc-install-operator.adoc[leveloffset=+1]
+== Installing {product-short}
```

Comment:
"Installing" is for the full journey: install the operator + create an operand.

Install the operator is a separate procedure, because it requires admin privileges.

"Deploying" stands here for "creating an Operator instance" / "create an operand". A developer can do this.

This might need refinement in a next PR.